onnxruntime/contrib_ops/cpu/bert/attention.cc

```diff
@@ -179,7 +179,6 @@ template <typename T>
 Attention<T>::Attention(const OpKernelInfo& info) : OpKernel(info), AttentionCPUBase(info) {
 }
 
-
 template <typename T>
```

Comment:
>templ [](start = 0, length = 5)

Stray change?

Reply:
Yes, unfortunately it seems like a stray single line change crept through. I ll just merge it instead of wasting CI resources for this.